tests/core/gui/test_file_history.py

```diff
@@ -60,4 +60,4 @@ def _history_displayed():
         assert hb2_w.label_user.text() == "Boby McBobFace"
         assert hb2_w.label_version.text() == "2"
 
-    await aqtbot.wait_until(_history_displayed)
+    await aqtbot.wait_until(_history_displayed, timeout=3000)
```

Comment:
why a specific timeout here ? This is most likely a code smell...

Reply:
It's just that the history is very slow at the moment.

Reply:
can't wait for #1407 to be done :trollface: